content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -38,6 +38,10 @@ The following are typical use cases for Deployments:
 
 ## Creating a Deployment
 
+Before creating a deployment refer on how to define an 
```

Comment:
- Do you know the difference between "deployment" and "Deployment", i.e.
  Why/when we are using "Deployment" instead of "deployment"?
- Is "refer on how to ..." good English?

/approve cancel